low-level/framework/utils/utility.py

```diff
@@ -39,8 +39,9 @@ def execute_cmd(self, command=[]):
         """
         #TODO Handle exception at caller side
         process = subprocess.Popen(command, shell=False, stdout=subprocess.PIPE,\
-                                    stderr=subprocess.PIPE)
-        return process.communicate()[0], process.returncode
+                                   stderr=subprocess.PIPE)
+        _result, _err = process.communicate()
+        return _result, _err, process.returncode
```

Comment:
Does this need changes at other places where this utility class used?

Reply:
This is getting used only in my patch for now. So, no change required

Reply:
ok